Fixed an invalid string value issue in the GreptimeDB connector following #16121
